### ISSUE TITLE
Training From Scratch 

### ISSUE THREAD

**Issue by mu745511** [david8862/keras-YOLOv3-model-set#212]
Hey @david8862 thanks for the work i was training yolov3 with darknet53 backbone without any pretrained weights , i trained for 30 epochs on voc2007 dataset but my class scores rarely reached 0.1 ( it was very low ) and predictions was bad i know that if i set my score threshold to 0.001 or lower i will get good map but i dont want false positive can you tell me how much epoch it took you to train the model from scratch ( without any pretrained weights ) and after how many epochs you started looking at good preds , my loss are converging but class probability are not increasing . my train config :- optimizer = SGD  , initial_lr = 0.001, scheduler = inverse_time_decay ( default args ) , image resolution = 512 , anchors = resized for 512 image reso .

**Comment by david8862** [comment]
@mu745511 I've not tried to train from scratch and not suggest that, especially on VOC dataset. If you do want to have a try maybe you can start from COCO, which has much more samples.

**Comment by mu745511** [comment]
@david8862 thanks for giving me your advice can you also tell me about  how much epoch will be enough for yolov3 with darknet or xceptione net backbone ? like 200-300 epochs ? i havent trained any model from scratch without any pretrained weights , Optimizer like SGD is helping me atm for 1e-3 initial lr as other optimizers seems to give me nan loss values .

**Comment by david8862** [comment]
@mu745511 as I know, official YOLOv3/v4/v5 repo usually apply 250~300 epochs in training config, so I set 250 as default value in train.py

**Comment by mu745511** [comment]
hey @david8862  thanks for the info !!!
